Doc/reference/compound_stmts.rst

```diff
@@ -312,11 +319,17 @@ when leaving an exception handler::
    >>> print(sys.exc_info())
    (None, None, None)
 
+
 .. index::
    keyword: except_star
 
-The :keyword:`except*<except_star>` clause(s) are used for handling
-:exc:`ExceptionGroup`\ s. The exception type for matching is interpreted as in
+.. _except_star:
+
+:keyword:`!except*` clause
+--------------------------
+
+The :keyword:`except* <except_star>` clause(s) are used for handling
+:exc:`ExceptionGroup`\s. The exception type for matching is interpreted as in
 the case of :keyword:`except`, but in the case of exception groups we can have
 partial matches when the type matches some of the exceptions in the group.
 This means that multiple except* clauses can execute, each handling part of
```

Comment:
```suggestion
This means that multiple :keyword:`!except*` clauses can execute, each handling part of
```
There's another at the end of this paragraphs that I can't reach.  These however might be out of scope for this PR, so feel free to create a separate one (there are also a few other keywords that should either get a `!` or that have no markup)..

Reply:
Ah, yup, thanks—I went ahead and fixed the keywords modified by this PR in the subsections it touched, of which there were quite a few (and required some rewrapping to keep below 80 columns).